jetty-webapp/src/test/resources/alt-jsp-webdefault.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can this file be simpler? Do we need all this?

Reply:
We need this to trigger the conditions.
Well, we need a default descriptor that doesn't have "jsp" or "default" defined.
It can probably be pruned a great deal, but it should still exist.